features/03_testing_frameworks/cucumber/steps/command/check_output_of_command.feature

```diff
@@ -179,22 +201,32 @@ Feature: All output of commands which were executed
     When I run `cucumber`
     Then the features should all pass
 
-  Scenario: Detect subset of one-line output
+  Scenario: Failed detection of exact multi-line output
     Given an executable named "bin/aruba-test-cli" with:
     """bash
     #!/usr/bin/env bash
 
-    echo 'hello world'
+    echo -e "goodbye\nworld"
     """
     And a file named "features/output.feature" with:
     """cucumber
     Feature: Run command
       Scenario: Run command
         When I run `aruba-test-cli`
-        Then the output should contain "hello world"
+        Then the output should contain exactly:
+        \"\"\"
+        hello
+        world
+        \"\"\"
     """
     When I run `cucumber`
-    Then the features should all pass
+    Then the features should not all pass with:
+    """
+          expected `aruba-test-cli` to have output output string is eq: "hello\nworld"
```

Comment:
...but now I understand there's more to it, we're surfacing RSpec matchers more directly in these messages.

Reply:
Please disregard my first comment.